modules/server/src/main/scala/navigate/server/tcs/TcsEpicsSystem.scala

```diff
@@ -53,12 +53,14 @@ object TcsEpicsSystem {
     val focusOffsetCmd: Command1Channels[F, Double]
     val oiwfsProbeTrackingCmd: ProbeTrackingCommandChannels[F]
     val oiwfsProbeCmds: ProbeCommandsChannels[F]
+    val m1GuideConfigCmd: Command4Channels[F, String, String, Int, String]
```

Comment:
Unrelated but could some of these be shared with `Observe`?

Reply:
Yes, that is the plan. But it will be a major overhaul of the *Epics clases in Observe.